tests/resource/model_resource.py

```diff
@@ -3,24 +3,97 @@
 
 .. moduleauthor:: Peter Morihladko <morihladko@atteq.com>
 """
-from zsl.unittest import TestCase
+from unittest import TestCase
 
+from zsl import Zsl
 from zsl.resource.model_resource import ModelResource
-from tests.resource.resource_test_helper import DummyModel, create_resource_test_data
+from tests.resource.resource_test_helper import DummyModel,\
+    create_resource_test_data, dummies, DummyTuple, DummyAppModel,\
+    test_settings
 
 
-class TestModelResource(TestCase):
+class TestRestResource(ModelResource):
+    __model__ = DummyModel
 
-    @classmethod
-    def setUpClass(cls):
-        create_resource_test_data()
 
+class ModelResourceTest(TestCase):
```

Comment:
Fixed, I try to group lines which handles similar things like "reading from file, computing something, closing", but when there's only one line per action it looks really spread out.